notebooks/mg-20230513-00-embedding-ideas.ipynb

```diff
@@ -0,0 +1,768 @@
+{
```

Comment:
<p>Since we're already here, could you just clean up this notebook by stripping out all the code/notes above that isn't necessary?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dsgt-birdclef/birdclef-2023/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>will do it!</p>